lib/pry-doc/pry_ext/show_source_with_c_internals/code_fetcher.rb

```diff
@@ -0,0 +1,125 @@
+require 'fileutils'
+require_relative 'c_file'
+require_relative 'symbol_extractor'
+
+module Pry::CInternals
+  class CodeFetcher
+    include Pry::Helpers::Text
+
+    class << self
+      attr_accessor :ruby_source_folder
+    end
+
+    # normalized
+    def self.ruby_version() RUBY_VERSION.tr(".", "_") end
+
+    self.ruby_source_folder = File.join(File.expand_path("~/.pry.d"), "ruby-#{ruby_version}")
+
+    attr_reader :line_number_style
+    attr_reader :symbol_extractor
+
+    def initialize(line_number_style: nil)
```

Comment:
I think `nil` here is implied by default, so you can just write `def initialize(line_number_style:)`.